### ISSUE TITLE
Statevector drawer - `state_to_latex` latex output contains `\rangleNone`

### ISSUE THREAD

**Issue by rishacha** [Qiskit/qiskit-terra#8169]
### Environment

- **Qiskit Terra version**: `0.20.2`
- **Python version**:  `Python 3.10.5 (main, Jun 11 2022, 16:53:24) [GCC 9.4.0] on linux`
- **Operating system**: `Linux - Elementary OS`

### What is happening?

When I try to render the following state - 

```py
sv = Statevector([ 7.07106781e-01-8.65956056e-17j,
             -5.55111512e-17-8.65956056e-17j,
              7.85046229e-17+8.65956056e-17j,
             -7.07106781e-01+8.65956056e-17j,
              0.00000000e+00+0.00000000e+00j,
             -0.00000000e+00+0.00000000e+00j,
             -0.00000000e+00+0.00000000e+00j,
              0.00000000e+00-0.00000000e+00j],
            dims=(2, 2, 2))
```

using  `sv.draw('latex')`

I get the following latext output containing - `\rangleNone` which is **not the intended behaviour**

![image](https://user-images.githubusercontent.com/30197357/173353308-bb33db27-3dc6-463f-b1ab-0e4f42125ec6.png)

### How can we reproduce the issue?

Code to reproduce the issue -

```py
from qiskit.quantum_info import Statevector
sv = Statevector([ 7.07106781e-01-8.65956056e-17j,
             -5.55111512e-17-8.65956056e-17j,
              7.85046229e-17+8.65956056e-17j,
             -7.07106781e-01+8.65956056e-17j,
              0.00000000e+00+0.00000000e+00j,
             -0.00000000e+00+0.00000000e+00j,
             -0.00000000e+00+0.00000000e+00j,
              0.00000000e+00-0.00000000e+00j],
            dims=(2, 2, 2))
sv.draw('latex')
```

### What should happen?

Expected output should be like this : $$\frac{1}{\sqrt{2}}|000\rangle -  \frac{1}{\sqrt{2}}|011\rangle$$

### Any suggestions?

One can fix it by looking at this particular patch of code - In `state_visualization.py`, inside function `_state_to_latex_ket`

https://github.com/Qiskit/qiskit-terra/blob/5d1205a84a47bec4d9d41be0215a9a0ae93264bc/qiskit/visualization/state_visualization.py#L1243-L1253

**Comment by jay-tau** [comment]
Hello, I'd love to work on this! Can you assign me to it, please? Thank you!

**Comment by diegoaichele** [comment]
@jay-tau I think this issue is being resolved in pull #8273

**Comment by rishacha** [comment]
Try looking at #8317. That's a follow up
